### DISCUSSION TITLE
12 New Components

### DISCUSSION THREAD

**Discussion by nik-rev** [shadcn-ui/ui#4610] [General]
There are quite a lot of open PRs in this repo and some of them have really useful components. While they aren't merged yet, we can still use the component in our projects. I'm sharing this with others, maybe some people will find this useful!

### New components

| Code | PR |
|----------|----------|
| [stepper](https://github.com/shadcn-ui/ui/blob/107224853495d698ccd4c665bae699d97a7dd6fb/apps/www/registry/new-york/ui/stepper.tsx) | https://github.com/shadcn-ui/ui/pull/318 |
| [typography](https://github.com/shadcn-ui/ui/blob/94ee191d989cf93246f2feaca68b1fdb24c18940/apps/www/registry/default/ui/typography.tsx) | https://github.com/shadcn-ui/ui/pull/11 |
| [timeline](https://github.com/shadcn-ui/ui/blob/53fc16bd875a161ea1fe9d9cd22d9d03c4cd8d8a/apps/www/registry/default/ui/timeline.tsx) | https://github.com/shadcn-ui/ui/pull/3374 |
| [calendar](https://github.com/shadcn-ui/ui/blob/97557f3d11909b8a1dff9c83fe7aaa89fc92a549/apps/www/registry/default/ui/calendar.tsx) | https://github.com/shadcn-ui/ui/pull/4421|
| [multi-select](https://github.com/shadcn-ui/ui/blob/844f1313f9a1cb5d6d02fcb47978869ffd3d7563/apps/www/registry/default/ui/multi-select.tsx)|https://github.com/shadcn-ui/ui/pull/2773|
|[input-number](https://github.com/shadcn-ui/ui/blob/cfcc165f299bf4f24012bf2c9a7ce012d5bb1a11/apps/www/registry/default/ui/input-number.tsx)|https://github.com/shadcn-ui/ui/pull/1818|
|[spinner](https://github.com/shadcn-ui/ui/blob/03021d26dc3791a1b52bb8a4ad065cb0ae58f92f/apps/www/registry/default/ui/spinner.tsx)| https://github.com/shadcn-ui/ui/pull/1583|
|[avatar-group](https://github.com/shadcn-ui/ui/blob/1adff7798e4a1ec635e4785a878f6c8ef4cf35d3/apps/www/registry/default/ui/avatar-group.tsx)| https://github.com/shadcn-ui/ui/pull/2474 |
| [rating](https://github.com/shadcn-ui/ui/pull/2350/files#diff-3d8583627d54747232d5a2a3ed3f57c04c377c5f3878300c11536c9281c7f51c)|https://github.com/shadcn-ui/ui/pull/2350|
|[selector](https://github.com/shadcn-ui/ui/blob/d7508e918f48c26535812645e4309ae35d41e8be/apps/www/registry/default/ui/selector.tsx)|https://github.com/shadcn-ui/ui/pull/2242|
|[password-compare](https://github.com/shadcn-ui/ui/blob/8dd00ddca730d332c2a6ec48656fcbb86b53cbfc/apps/www/__registry__/default/ui/password-compare.tsx)|https://github.com/shadcn-ui/ui/pull/3452|
|[container](https://github.com/shadcn-ui/ui/blob/36e21074b10ddac33c8b80b5603286138752b768/apps/www/__registry__/default/ui/container.tsx)|https://github.com/shadcn-ui/ui/pull/4079|

**Comment by stefandevo** [comment]
I Completely do not understand why this project is not maintained or followed up.. 600+ prs not looked at apparently. Is it time to fork it?

**Comment by its-monotype** [comment]
Here is another implementation of the Rating component that appears to be more feature-rich: https://github.com/shadcn-ui/ui/pull/2350

**Reply by nik-rev** [reply]
> Here is another implementation of the Rating component that appears to be more feature-rich: #2350

That is cool, looks like the most important improvement here is that icons can have a % fill. Thanks for sharing!
